easybuild/easyconfigs/s/spaln/spaln-2.4.12-GCC-10.2.0.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'spaln'
+version = '2.4.12'
+
+homepage = 'https://github.com/ogotoh/spaln'
+description = """Spaln (space-efficient spliced alignment) is a stand-alone program that maps
+ and aligns a set of cDNA or protein sequences onto a whole genomic sequence in a single job."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+# disable use of -march=native, which makes compilation fail due to missing header files like fwd2s1_simd.h;
+# see also https://github.com/ogotoh/spaln/issues/56
+toolchainopts = {'optarch': False}
+
+source_urls = ['https://github.com/ogotoh/spaln/archive/']
+sources = ['ver%(version)s.tar.gz']
+checksums = ['0c5f842131409686dbaad01778960f53f1ebba8dd83d55b454ec4bf1b6bde2af']
+
+dependencies = [('zlib', '1.2.11')]
+
+start_dir = 'src'
+
+# we need to make sure not to pass --prefix, otherwise the configure script gets stuck in an infinite loop...
+prefix_opt = '--exec_prefix='
+
+configopts = "--exec_prefix=%(installdir)s/bin --table_dir=%(installdir)s/table --alndbs_dir=%(installdir)s/seqdb "
+configopts += "--use_zlib=1"
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['makdbs', 'makmdm', 'sortgrcd', 'spaln']],
+    'dirs': ['seqdb', 'table'],
+}
+
+sanity_check_commands = ["spaln -h 2>&1 | grep 'SPALN version %(version)s'"]
+
```

perhaps we should also add the scripts to `PATH`
```suggestion

modextrapaths = {'PATH': 'seqdb'}

```

---

fixed in 83e4d44